_posts/2021-09-20-logicdrop-customer-story.adoc

```diff
@@ -0,0 +1,193 @@
+
+* It's too hard to setup the environment
+* The learning curve was too steep
+* Our CI/CD process was automated but not enough
+
+Enter 2021. Our goal is not just to get back to status quo, it's to introduce major improvements to our process and make the jump head-first into becoming cloud-native!
+
+== WTF is Quarkus?
+
+Like the https://quarkus.io[website^] says, it really is pure *developer joy*. 
+
+Before jumping into Quarkus we evaluated many stacks, Spring Boot, Micronaut, even Guice was a consideration at one point. We also looked at more drastic changes like switching over to Python or .NET Core.
+
+At the end of the day, Quarkus won us over because it is the Java, we all know and love, but on steriods when needed, and was:
+
+* Quickly picked-up by developers
+* The easiest to become productive with
+* A rich and cohesive ecosystem
+* Able to handle reactive or imperative naturally
+* Kubernetes and Knative friendly
+* Based on standards we were already familiar with
+
+The deeper we explored Quarkus, the more we began to appeciate other aspects: 
+
+* https://quarkus.io/guides[Guides^] that were simple enough to get started but applicable to real-world tasks
+* A super responsive and helpful https://quarkus.io/community[community^], especially the core developers
+
+At this point we decided to take the plunge but, rather than retrofit our existing platform, we would start from a clean slate. At the same time, we would push the envelope further and completely overhaul our CI/CD process, clusters, and most importantly, our developer culture.
+
+The lead architect responsible for prototyping says:
+
+[quote]
+____
+In less than a month I've already implemented and deployed at least three new significant prototypes using Quarkus. I can't believe two of them have been rejected already.
+
+*Lead Architect*
+____
+
+== Change is Inevitable
+
+Since we approached this as a ground up rewrite of our existing Spring Boot platform there were many changes that needed to happen.  The codebase was new but patterns were the same and using Quarkus we were able to implement them easier, faster, and better.
+
+* VSCode was able to become our defacto editor
+* AutoConfigurations and wasted dependencies went away (big win)
+* DI became predictable not magical (big win)
+* Using standards made development, and learning, the codebase easier
+* Mongo and Panache did just what we needed nothing more
+* Kubernetes configurations could be kept closer to the service
+* Mutiny made writing reactive services way easier for everyone
+* No more reflection to scan "beans" to help configuration or DI
+* Security worked out-of-the-box and was well defined (big win)
+* REST clients helped isolate services cleaner and better
+* OpenAPI and Swagger worked without tweaking
+* Intimate knowledge of Maven was no longer required
+* Testing became easier and consistent
+* CI/CD was responsible across all services rather than individually
```

Comment:
Responsible ? Do you mean reproducible or repeatable ? Feels like a word missing ?